engine/src/main/java/org/terasology/engine/world/viewer/layers/FacetLayers.java

```diff
@@ -78,7 +78,7 @@ private static Collection<FacetLayer> createLayersFor(Class<? extends WorldFacet
         }
 
         if (result.isEmpty()) {
-            logger.warn("No layers found for facet {}", facetClass.getName());
+            logger.atWarn().addArgument(() -> facetClass.getName()).log("No layers found for facet {}");
```

Comment:
Simple getter for member access, neglectable impact, ignore PMD warning
```suggestion
            logger.warn("No layers found for facet {}", facetClass.getName());  //NOPMD
```